docs/en/integrations/language-clients/java/index.md

```diff
@@ -545,6 +545,21 @@ For a detailed guide, go to [Configuring HTTP library](/docs/en/integrations/jav
 
 Alternatively, you can add equivalent parameters to the JDBC URL.
 
+The JDBC client default connection timeout and socket timeout are 30 seconds. The timeout can be increased to support large data insert operations. Use the `options` method on `ClickHouseClient` together with the `SOCKET_TIMEOUT` and `CONNECTION_TIMEOUT` options as defined by `ClickHouseClientOption`:
+
+```java
+int MS_12H = 43_200_000; // 12 hours in milliseconds
```

Comment:
Can you change it to 12 * 60 * 60 * 1000

Reply:
Fixed now.